demos/ASP.Net Core/Divergent.CompositionGateway/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:4457/",
+      "sslPort": 0
+    }
+  },
+    "profiles": {
+        "IIS Express": {
+            "commandName": "IISExpress",
+            "launchBrowser": true,
+            "environmentVariables": {
+                "ASPNETCORE_ENVIRONMENT": "Development"
+            }
+        },
+        "Divergent.CompositionGateway": {
+            "commandName": "Project",
+            "launchBrowser": true,
+            "environmentVariables": {
+                "ASPNETCORE_ENVIRONMENT": "Development"
+            },
+            "applicationUrl": "http://localhost:4458"
+        }
+    }
+}
```

Comment:
it's managed by VS when you touch the project property page, I'll fix it, I'm not sure it won't be changed once more.